js/src/annotations/osd-svg-rectangle.js

```diff
@@ -38,7 +37,7 @@
       return shape;
     },
 
-    _createSegments:function(initialPoint,pixel, overlay){
+    _createSegments:function(initialPoint, overlay){
```

Comment:
I assume changing the arguments for this "private" method is acceptable as a non-api breaking change?